baybe/utils/dataframe.py

```diff
@@ -420,7 +420,7 @@ def fuzzy_row_match(
     return pd.Index(inds_matched)
 
 
-def pretty_print_df(df: pd.DataFrame) -> str:
+def pretty_print_df(df: pd.DataFrame, max_rows: int = 6, max_columns: int = 4) -> str:
     """Convert a dataframe into a pretty/readable format.
 
     This function returns a string representation containing at most the
```

Comment:
the docsting here still says `This function returns a string representation containing at most the
    first 10 rows and 7 columns of the dataframe.` which doenst make sense because these numbers are configurable, or is this truly a limit of pandas??